index.php

```diff
@@ -110,6 +111,7 @@
 				throw new Exception("Invalid password! <a href='./password_reset.php'>Forgot password?</a>");
 			}
 		} catch (Exception $e) {
+            $system->rollbackTransaction();
 			$system->message($e->getMessage());
```

Comment:
```suggestion
			$system->message($e->getMessage());
			error_log($e->getMessage());
```

Reply:
Wonder if we should roll this into one system method, like we have `$system->error` right now, maybe it works or maybe we make a new method that just calls the three lines

```php
$system->rolbackTransaction();
error_log($error_message);
$system->message($error_message);
```